### ISSUE TITLE
Regression: panic on CLI login

### ISSUE THREAD

**Issue by kontsevoy** [gravitational/teleport#721]
This is a regression, I suspect it's caused by totp 2nd factor auth.
Executing `tsh ssh` on a new cluster, I get:

```
turing /var/go/src/github.com/gravitational/teleport: tsh --proxy=localhost ssh localhost 
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x4a5f09]

goroutine 1 [running]:
panic(0x8d6220, 0xc420010100)
	/opt/go/src/runtime/panic.go:500 +0x1a1
github.com/gravitational/teleport/lib/client.(*TeleportClient).Login(0xc4201ae000, 0xc4201c2420, 0xc42016d401)
	/home/ekontsevoy/go/src/github.com/gravitational/teleport/lib/client/api.go:973 +0x119
github.com/gravitational/teleport/lib/client.(*TeleportClient).ConnectToProxy(0xc4201ae000, 0x0, 0x0, 0x0)
	/home/ekontsevoy/go/src/github.com/gravitational/teleport/lib/client/api.go:905 +0x45d
github.com/gravitational/teleport/lib/client.(*TeleportClient).SSH(0xc4201ae000, 0xdf7e40, 0xc4200102f8, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
	/home/ekontsevoy/go/src/github.com/gravitational/teleport/lib/client/api.go:410 +0x7a
main.onSSH(0xc42009db80)
	/home/ekontsevoy/go/src/github.com/gravitational/teleport/tool/tsh/main.go:301 +0xcd
main.run(0xc42000c3d0, 0x3, 0x3, 0xc420000100)
	/home/ekontsevoy/go/src/github.com/gravitational/teleport/tool/tsh/main.go:163 +0x323e
main.main()
	/home/ekontsevoy/go/src/github.com/gravitational/teleport/tool/tsh/main.go:40 +0x6b
```

This is happens inside of `switch tc.SecondFactorType`, i.e.

**Comment by kontsevoy** [comment]
Looks like `constants.go` had `teleport.HOTP` declared as "htop" instead of "hotp".

**Comment by kontsevoy** [comment]
closed by #722
